cli/azd/internal/scaffold/funcs.go

```diff
@@ -187,7 +187,7 @@ var camelCaseRegex = regexp.MustCompile(`([a-z0-9])([A-Z])`)
 // EnvFormat takes an input parameter like `fooParam` which is expected to be in camel case and returns it in
 // upper snake case with env var template, like `${AZURE_FOO_PARAM}`.
 func EnvFormat(src string) string {
-	snake := strings.ToUpper(camelCaseRegex.ReplaceAllString(src, "${1}_${2}"))
+	snake := strings.ReplaceAll(strings.ToUpper(camelCaseRegex.ReplaceAllString(src, "${1}_${2}")), "-", "_")
```

Comment:
Did we consider using [AlphaSnakeUpper](https://github.com/Azure/azure-dev/blob/88a51a039d6af636deacffaf1735b94a165281ef/cli/azd/internal/scaffold/funcs.go#L52C6-L52C21)?

Reply:
AlphaSnakeUpper removes characters like `-` , which I need to replace for `_`